src/hotspot/cpu/aarch64/macroAssembler_aarch64.cpp

```diff
@@ -4855,7 +4855,7 @@ address MacroAssembler::zero_words(Register ptr, Register cnt)
 // r10, r11, rscratch1, and rscratch2 are clobbered.
 address MacroAssembler::zero_words(Register base, uint64_t cnt)
 {
-  guarantee(zero_words_block_size < BlockZeroingLowLimit,
+  guarantee(wordSize <= BlockZeroingLowLimit,
```

Comment:
Surely, this `guarantee` would now never fire? Maybe we should demote it to `assert` to protect from accidental range error. There is no need to pay the price in release bits, I think.

Reply:
True.